lib/cartopy/mpl/slippy_image_artist.py

```diff
@@ -40,7 +40,7 @@ class SlippyImageArtist(AxesImage):
     def __init__(self, ax, raster_source, **kwargs):
         self.raster_source = raster_source
         super(SlippyImageArtist, self).__init__(ax, **kwargs)
-        self.set_clip_path(ax.background_patch)
+        self.set_clip_path(ax.patch)
```

Comment:
Now that the background_patch is in the axes, won't this artist be clipped automatically by your new `ax.patch`?

Reply:
You're right, I've remove this.